Jenkinsfile

```diff
@@ -1,5 +1,55 @@
 #!/usr/bin/env groovy
-common {
-  slackChannel = '#connect-eng'
-  upstreamProjects = 'confluentinc/common'
+
+/*
+Copied from jenkins-common/var/common.groovy
```

Comment:
what's the reason that you can't use `common` or `dockerfile` step? It would be good to list it out here for context and future readers. 

Maintaining this here is ok if this is a unique build workflow/steps pattern that nowhere else would use -- or rarely.

Reply:
@maxzheng, we actually expect this to be a common pattern for dozens of connectors. I will open a parallel PR into jenkins-common to add a new groovy function to handle this case. Just wanted to put it all here first to make sure it runs!